index.js

```diff
@@ -26,30 +24,30 @@ PacketStream.prototype.message = function (obj) {
 // Sends a message to the other end, expects an (err, obj) response
 PacketStream.prototype.request = function (obj, cb) {
   if (this._closing) return cb(new Error('parent stream is closing'))
-  var rid = this._req_counter++
-  var self = this
```

Comment:
The `self` pattern was removed, because if you use `() => {}` functions (line 28), the `this` never refers to `() => {}`

Reply:
Great, didn't know that. I always get confused about `this`, glad they cleaned that up a bit with () => {}